### ISSUE TITLE
a suggestion

### ISSUE THREAD

**Issue by mrose17** [sibartlett/homebridge-wink3#20]
perhaps the configuration options for `hide_ids` and `window_ids` could take a string like `"sensor_pod/123"` in addition to `"123"` ... that would remove any ambiguity for different device types that have the same numeric-identifier...

**Comment by sibartlett** [comment]
Yes, I agree. Although it's unlikely that a user will have 2 device types with the same numeric-identifier, it's still possible. I'll consider implementing this soon.

**Comment by mrose17** [comment]
Thanks!  Great plugin, by the way!

/mtr
________________________________
